shm_wrapper/shm_start.c

```diff
@@ -127,14 +127,17 @@ int main() {
         dev_shm_ptr->net_sub_map[i] = -1;
         dev_shm_ptr->exec_sub_map[i] = 0;
     }
-    gp_shm_ptr->buttons = 0;
-    for (int i = 0; i < 4; i++) {
-        gp_shm_ptr->joysticks[i] = 0.0;
+    for (int j = 0; j < 2; j++) {
```

Comment:
actually thinking about it, having more than 2 sources would break a lot of the code. so I don't see the need to have a define statement. similar to only having data and command streams for devices